geoserver/webapp/src/docker/docker-entrypoint.d/01-copy-jetty-env-if-found

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+FILE=/etc/georchestra/geoserver/jetty-env.xml
```

Comment:
`/etc/georchestra` is purely conventional, don't we have an env variable which could be used instead of the hardcoded path ?

Reply:
actually it is also set / hardcoded into the Dockerfile, so I'm ok with the change:
https://github.com/georchestra/georchestra/blob/master/geoserver/webapp/src/docker/Dockerfile#L42